racket/collects/racket/private/class-internal.rkt

```diff
@@ -331,8 +331,9 @@
               (append
                (kernel-form-identifier-list)
                (list
-                (quote-syntax #%app) ; racket/base app, as opposed to #%plain-app
+                (quote-syntax #%app) ; racket/base #%app, as opposed to #%plain-app
```

Comment:
Please revert all changes to this file. `free-identifier=?` already takes renaming into account. There's no need to test against another renamed form.

Reply:
This file already handles similar processes for `λ`, as seen in: `(or (free-identifier=? #'lam #'lambda) (free-identifier=? #'lam #'λ))`. I assumed the relationship between `lambda` and `λ` was similar to that between `case-lambda` and `case-λ`, which is why I treated `#%plain-λ` and `case-λ` in the same manner. If supporting `case-λ` is considered, do we still need to revert all changes to this file?

Reply:
I read the code, and if I understand correctly, `lambda` and `λ` in `racket/base` are respectively `new-lambda` and `new-λ` in `kw.rkt`:
```
   (define-syntaxes (new-lambda new-λ)
     (let ([new-lambda ...])
       (values new-lambda new-lambda)))
```
Is it necessary to create a new PR and define `new-λ` through the renaming mechanism?

Reply:
For any big change like this (including this PR actually), I think it makes sense to get preliminary feedback from https://racket.discourse.group/ first. For example, it could be that people are somehow relying on `lambda` and `λ` not being `free-identifier=?` for some reasons, and the change would break their code.

But I agree that _ideally_ `lambda` should be `free-identifier=?` to `λ`.

Reply:
It seems unlikely that changing how `lambda` and `λ` are bound is going to work but before a discussion, just checking to see if racket (and a bunch of packages, maybe starting with main-distribution) builds would be informative.